src/validations/member.js

```diff
@@ -0,0 +1,28 @@
+const Joi = require('joi');
+
+const validateMembersUpdate = (req, res, next) => {
+  const membersUpdate = Joi.object({
+    firstName: Joi.string().min(3).max(25).required(),
```

Comment:
I would add a validation to prevent the user from entering numbers when dealing with strings

Reply:
The requested changes were done!..